src/api/pastel-rpc/rpc.ts

```diff
@@ -29,15 +30,30 @@ export async function rpc<T>(
     })
   } catch (err) {
     if (err.response) {
+      log.error(
```

Comment:
please log.error to console.error as now electron log is assigned to a `console`, and in other places in this file as well

Reply:
Fixed